spec/system/teacher_interface/authentication_spec.rb

```diff
@@ -189,26 +189,26 @@ def when_i_click_save_and_sign_out
   end
 
   def then_i_see_the_sign_up_form
-    expect(page).to have_title("Your email address")
-    expect(page).to have_content("Your email address")
-    expect(page).to have_content(
+    expect(sign_up_page.heading).to have_content("Your email address")
```

Comment:
I think with these where we know the exact text and it's not that long we should use `.text` as it's more explicit:
```
expect(sign_up_page.heading.text).to eq("Your email address<etc>...")
```